packages/imitation_game/imitation_tests/smiley/android/run_android.sh

```diff
@@ -0,0 +1,14 @@
+set -e
+cd $( dirname "${BASH_SOURCE[0]}" )
+source ../../../helper/run_android_helper.sh
+
+readonly BUNDLE="dev.flutter.imitation_game.smiley"
+readonly TEST_NAME="smiley"
+
+# TODO(gaaclarke): Get assembleRelease working.  I don't know if the performance
+# is any different but assembleRelease requires signing the apk.
+./gradlew assembleDebug
+launch_apk ./app/build/outputs/apk/debug/app-debug.apk $BUNDLE "MainActivity"
+sleep 10
```

Comment:
you can probably stream adb logcat for a success/failure result. @blasten might have other ideas too.